spec/ruby/core/module/attr_spec.rb

```diff
@@ -33,66 +33,6 @@ def initialize
     o.send(:attr3).should == "test3"
   end
 
-  it "creates a setter for the given attribute name if writable is true" do
```

Comment:
Specs should not be removed, instead they should be under `ruby_version_is ""..."3.1" do` guard.

Reply:
OK, I'll fix it.

Reply:
Looks good now, but there should be a new `it` spec with `ruby_version_is "3.1" do` describing the new behavior, which likely is just some kind of error.